python/lsst/obs/decam/decamMapper.py

```diff
@@ -221,7 +245,20 @@ def std_raw(self, item, dataId):
         return self._standardizeExposure(self.exposures['raw'], exp, dataId,
                                          trimmed=False)
 
-    def _standardizeMasterCal(self, datasetType, item, dataId, setFilter=False):
+    def std_dark(self, item, dataId):
```

Comment:
This code was added in 756fa7f75. In that commit, and in 4510e6e which followed it, your attribution has regressed from "Hsin-Fang Chiang <hchiang2@illinois.edu>" to "hchiang2 <hchiang2@illinois.edu>". Obviously it's entirely up to you what you call yourself, but I suggest the former is what you were aiming for!